pkgs/applications/misc/blender/derivation.nix

```diff
@@ -0,0 +1,280 @@
+{ config, stdenv, lib, fetchFromGitea, fetchzip, boost, cmake, ffmpeg, gettext, glew
+, ilmbase, libepoxy, libXi, libX11, libXext, libXrender
+, libjpeg, libpng, libsamplerate, libsndfile
+, libtiff, libwebp, libGLU, libGL, openal, opencolorio, openexr, openimagedenoise, openimageio, openjpeg
+, openvdb, libXxf86vm, tbb, alembic
+, zlib, zstd, fftw, opensubdiv, freetype, jemalloc, ocl-icd, addOpenGLRunpath
+, jackaudioSupport, libjack2
+, cudaSupport, cudaPackages ? { }
+, hipSupport, rocmPackages # comes with a significantly larger closure size
+, colladaSupport, opencollada
+, spaceNavSupport, libspnav
+, makeWrapper
+, pugixml, llvmPackages, SDL, Cocoa, CoreGraphics, ForceFeedback, OpenAL, OpenGL
+, waylandSupport, pkg-config, wayland, wayland-protocols, libffi, libdecor, libxkbcommon, dbus
+, potrace
+, openxr-loader
+, embree, gmp, libharu
+, openpgl
+, mesa
+, runCommand
+, version, hashes, isLTS, pythonPackages, withPackages, xvfb-run
+}:
+
+let
+  python = pythonPackages.python;
+  optix = fetchzip {
+    # url taken from the archlinux blender PKGBUILD
+    url = "https://developer.download.nvidia.com/redist/optix/v7.3/OptiX-7.3.0-Include.zip";
+    sha256 = "0max1j4822mchj0xpz9lqzh91zkmvsn4py0r174cvqfz8z8ykjk8";
```

Comment:
Please use Sri hashes

Reply:
This was part of the original file. I'll change it though